src/foam/u2/view/CardSelectionView.js

```diff
@@ -58,6 +58,11 @@ foam.CLASS({
       name: 'choiceView',
       value: { class: 'foam.u2.view.CardSelectView' }
     },
+    {
+      class: 'FObjectProperty',
+      of: 'foam.mlang.predicate.Predicate',
+      name: 'selectionPredicate'
```

Comment:
Please add documentation:

Reply:
If this defaulted to FALSE, then you would need the null check below.

Reply:
it's undefined by default the optional chaining operator should to able take care of it.